common/services/population.js

```diff
@@ -18,6 +20,34 @@ const populationService = {
       include: ['moves_from', 'moves_to', 'location'],
     })
   },
+
+  create(locationId, date, data) {
```

Comment:
You might be able to simplify these methods by just passing data in similar to how the [moves service works](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/07b8b4dc197b790a985f63944b08de81eb5b1087/common/services/move.js#L239-L253).

You can still do some checks to ensure an ID is present but maybe just a property within the `data` argument.

Reply:
Refactored to use the move service approach